artemis-core-client/src/main/java/org/apache/activemq/artemis/core/message/impl/CoreMessage.java

```diff
@@ -426,7 +426,6 @@ protected CoreMessage(CoreMessage other, TypedProperties copyProperties) {
       // with getEncodedBuffer(), otherwise can introduce race condition when delivering concurrently to
       // many subscriptions and bridging to other nodes in a cluster
       synchronized (other) {
-         this.body = other.body;
```

Comment:
I might be a plank, but surely this avoids using up memory as the body object is copied by reference here, rather than later needing to deserialise again and creating a new object in memory.

Reply:
as well as a perf improvement as serialisation is a time burn.

Reply:
That body param seems was unused, I just clean it up as dead code